Packages/Tracking/Hands/Runtime/Scripts/HandBinder/HandBinder.cs

```diff
@@ -50,6 +50,10 @@ public class HandBinder : HandModelBase
         [Tooltip("Set the assigned transforms to the same position as the Leap Hand")]
         public bool SetPositions = true;
 
+
+        [Tooltip("Moves the elbow so that when the forearm scales the bone doesnt clip into the hand model")]
+        public bool UseScaleToPositionElbow = true;
```

Comment:
Can this mention that it's particularly useful for non-skinned hands

and can we default to false, and enable on the skeleton hands prefab only plz?

Reply:
Changed and pushed